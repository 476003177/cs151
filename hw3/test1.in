signup
username=fred
password=123

signup
username=wilma
password=letmein

signup
username=bozotheclown
password=none

login
username=fred
password=123

newquestion
text=How do I open a terminal window?

good
id=0
type=question

getquestion
id=0

login
username=wilma
password=letmein

getquestion
id=0

editquestion
id=0
text=How do I open a terminal window in Lubuntu?

answer
id=0
text=Hit Ctrl+Alt+T

login
username=horstmann
password=secret

good
id=0
type=studentanswer

answer
id=0
text=Or click on the bird icon and select Terminal from the menu

getquestion
id=0

getanswer
id=0
type=student

getanswer
id=0
type=instructor

login
username=bozotheclown
password=none

followup
id=0
text=Why do I need a terminal?

getfollowup
id=0
followupid=0

login
username=wilma
password=letmein

followupreply
id=0
followupid=0
text=So you can issue commands such as git pull

resolve
id=0
followupid=0

getfollowup
id=0
followupid=0

getfollowupreply
id=0
followupid=0
followupreplyid=0

