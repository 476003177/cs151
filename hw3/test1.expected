OK

OK

OK

OK

OK
id=0

OK

OK
text=How do I open a terminal window?
good=true
lastauthor=fred
views=1

OK

OK
text=How do I open a terminal window?
good=true
lastauthor=fred
views=2

OK

OK

OK

OK

OK

OK
text=How do I open a terminal window in Lubuntu?
good=true
lastauthor=wilma
views=3

OK
text=Hit Ctrl+Alt+T
good=true
lastauthor=wilma

OK
text=Or click on the bird icon and select Terminal from the menu
good=false
lastauthor=horstmann

OK

OK
followupid=0

OK
text=Why do I need a terminal?
author=bozotheclown
resolved=false

OK

OK
followupreplyid=0

OK

OK
text=Why do I need a terminal?
author=bozotheclown
resolved=true

OK
text=So you can issue commands such as git pull
author=wilma

